docs/data-sources/service_accounts.md

```diff
@@ -0,0 +1,43 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_service_accounts Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Services offer a way to provide programmatic, centrally-controlled, and consistent access controls. For more information, see Twingate's documentation https://www.twingate.com/docs/services.
```

Comment:
s/Services/Service Accounts/

Remove the sentence starting "For more information..."

Reply:
@alexmensch docs is updated